internal/cluster.go

```diff
@@ -351,8 +351,7 @@ func (cs *clusterService) GetMember(address core.Address) core.Member {
 	membersList := cs.members.Load().([]*protocol.Member)
 	for _, member := range membersList {
 		if *member.Address().(*protocol.Address) == *address.(*protocol.Address) {
-			copyMember := *member
-			return &copyMember
+			return member
```

Comment:
What is the reason for this change?

Reply:
Previous one seems more complex. We do not need to protect `Member`. We give it as interface to user.